ccd-definition/CaseTypeTab.json

```diff
@@ -26,9 +26,49 @@
     "TabID": "CasePeopleTab",
     "TabLabel": "People in the case",
     "TabDisplayOrder": 2,
-    "CaseFieldID": "caseIDReference",
+    "CaseFieldID": "children",
     "TabFieldDisplayOrder": 1
   },
+  {
+    "LiveFrom": 43101,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "Channel": "CaseWorker",
+    "TabID": "CasePeopleTab",
+    "TabLabel": "People in the case",
+    "TabDisplayOrder": 2,
+    "CaseFieldID": "respondents",
+    "TabFieldDisplayOrder": 3
+  },
+  {
+    "LiveFrom": 43101,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "Channel": "CaseWorker",
+    "TabID": "CasePeopleTab",
+    "TabLabel": "People in the case",
+    "TabDisplayOrder": 2,
+    "CaseFieldID": "enterApplicant_applicant",
```

Comment:
Would be great to get rid of `enterApplicant_` here and in definition row below. Not necessarily this PR though.

Reply:
ill just do it now